### PR TITLE
Application/multilingual rag

### PR DESCRIPTION
**Pull request by akashAD98**
added Multilingual rag app using cohere multilingual embeddings.with gradio ui

### CODE REVIEW

File: applications/Multilingual_RAG/main.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    "French": "fr",
+}
+
+# Logging Configuration
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def initialize_documents_and_embeddings(input_file_path):
+    file_extension = os.path.splitext(input_file_path)[1]
```
Comment (raghavdixit99) [inline applications/Multilingual_RAG/main.py:36]: Can you add docstrings to all the functions @akashAD98

**Review by raghavdixit99 — CHANGES_REQUESTED** [review]

File: applications/Multilingual_RAG/.env-example
[the remark this answers is not in the document]
Comment (AyushExel) [reply applications/Multilingual_RAG/.env-example:1]: Good idea but I think we won't be able to do this.. Some examples have conflicting dependencies so its best we treat each of them separately...

**Review by AyushExel — APPROVED** [review]

File: applications/Multilingual_RAG/main.py (hunk 1 above)
Comment (akashAD98) [reply applications/Multilingual_RAG/main.py:36]: okay ill add

Comment (akashAD98) [reply applications/Multilingual_RAG/main.py:45]: done

**Comment by PrashantDixit0** [comment]
I'll lint it properly 👍

**Comment by PrashantDixit0** [comment]
Done @akashAD98 Check your repo and update the PR :+1:

**Comment by akashAD98** [comment]
> Done @akashAD98 Check your repo and update the PR 👍

@PrashantDixit0 can you check now

**Comment by AyushExel** [comment]
Merging this but lets lint it later

**Pull request merged**